consensus/core/src/core_thread.rs

```diff
@@ -97,7 +99,8 @@ struct CoreThread {
     core: Core,
     receiver: Receiver<CoreThreadCommand>,
     rx_subscriber_exists: watch::Receiver<bool>,
-    rx_propagation_delay_and_quorum_rounds: watch::Receiver<(Round, Vec<QuorumRound>)>,
+    rx_propagation_delay_and_quorum_rounds:
+        watch::Receiver<(Round, Vec<QuorumRound>, Vec<QuorumRound>)>,
```

Comment:
nit: instead of a tuple maybe you could use a `struct` instead as now there are quite a few parameters and would be good to have better clarity on the semantics.